src/utils/fieldUtils/renderTooltip.js

```diff
@@ -10,7 +11,12 @@ export default function renderTooltip(tooltip, className, iconClassName) {
         right: true,
         ...overrides
       }),
-      icon: <Icon icon='tooltip' className={iconClassName} />
+      icon:
+  <Icon
```

Comment:
what rule led to this?

Reply:
No idea - weird linter change. I've changed it back.